Packages/com.unity.inputsystem/InputSystem/Plugins/Android/InputSettingsAndroid.cs

```diff
@@ -0,0 +1,49 @@
+#if UNITY_EDITOR || UNITY_ANDROID || PACKAGE_DOCS_GENERATION
+using System;
+
+namespace UnityEngine.InputSystem
+{
+    public partial class InputSettings
+    {
+        /// <summary>
+        /// Project-wide input settings for the Android platform.
+        /// </summary>
+        [Serializable]
+        public class AndroidSettings
+        {
+            /// <summary>
+            /// Determines whether pressing the back button on the device should leave the app or not.
+            /// </summary>
+            /// <remarks>
+            /// On Android 13 and above, you can use this option in combination with `PlayerSettings.Android.predictiveBackSupport`
+            /// to enable predictive back animations.
+            /// </remarks>
+            public bool backButtonLeavesApp
+            {
+                get
+                {
+                    return m_BackButtonLeavesApp;
+                }
+                set
+                {
+                    m_BackButtonLeavesApp = value;
```

Comment:
we should probably exit here without ApplySettings call if value matches m_BackButtonLeavesApp ?

```
if (value == m_BackButtonLeavesApp)
    return;
```

otherwise if you use it with stuff like IMGUI, you could end up calling ApplySettings every frame

Reply:
fee01d001061c0aab49e7bd8299f7103bd41b3b9